### DISCUSSION TITLE
Docker Image Support

### DISCUSSION THREAD

**Discussion by yjy44** [nocodb/nocodb#8321] [Q&A]
I checked the manual, and it appears that NocoDB creates a git image by cloning the git repository.

Are there any plans to create a docker image in advance and manage it on docker hub?

**Comment by mertmit** [comment]
Hi @yjy44,
While we use GitHub Actions to deploy every release to [Docker Hub](https://hub.docker.com/r/nocodb/nocodb), it's important to note that these images aren't merely clones of our git repository. Our Docker images are specifically tailored for production use, containing frontend and backend builds to ensure readiness for deployment.

For detailed instructions on running these Docker images, you can refer to [our documentation](https://docs.nocodb.com/getting-started/self-hosted/installation/#docker).

Let us know if you have any further questions!
